damus/Views/Wallet/ConnectWalletView.swift

```diff
@@ -178,7 +178,7 @@ struct ConnectWalletView: View {
             Text("Damus Wallet", comment: "Title text for Damus Wallet view.")
                 .fontWeight(.bold)
             
-            Text("Securely connect your Damus app to your wallet using Nostr Wallet Connect", comment: "Text to prompt user to connect their wallet using 'Nostr Wallet Connect'.")
+            Text("Securely connect your Damus app to your wallet using Nostr Wallet Connect", comment: "Text to prompt user to connect their wallet using 'Nostr Wallet Connect'.")
```

Comment:
@danieldaquino Using non-breaking spaces directly does indeed fix the issue. Re: https://github.com/damus-io/damus/pull/2755#issuecomment-2513963343

Reply:
Thank you @tyiu!!